src/Microsoft.Framework.ApplicationHost/Program.cs

```diff
@@ -29,6 +29,7 @@ public Program(IAssemblyLoaderContainer container, IApplicationEnvironment envir
 
         public Task<int> Main(string[] args)
         {
+            Logger.TraceInformation($"[{nameof(ApplicationHost)}] Launching DNX application on {_environment.RuntimeFramework}");
```

Comment:
Should remove, should try n keep eveyrthing as tiny as possible to remove places for possible errors. We can address this in a later issue/pr